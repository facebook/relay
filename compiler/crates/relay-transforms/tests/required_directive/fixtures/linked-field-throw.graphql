fragment EnabledActorsRendererFragment on User {
  id
  profile_picture @required(action: THROW) {
    uri
  }
}
