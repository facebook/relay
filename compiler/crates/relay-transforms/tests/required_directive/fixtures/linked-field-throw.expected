==================================== INPUT ====================================
fragment EnabledActorsRendererFragment on User {
  id
  profile_picture @required(action: THROW) {
    uri
  }
}
==================================== OUTPUT ===================================
fragment EnabledActorsRendererFragment on User {
  id
  profile_picture @required(action: THROW) @__required(action: "THROW", path: "profile_picture") {
    uri
  }
}
